src/erc3525/extensions/metadata/interface.cairo

```diff
@@ -0,0 +1,13 @@
+const IERC3525_METADATA_ID : felt252 = 0xe1600902;
+
+#[starknet::interface]
+trait IERC3525Metadata<TContractState> {
+    fn contract_uri(self: @TContractState) -> felt252;
+    fn slot_uri(self: @TContractState, slot: u256) -> felt252;
+}
+
+#[starknet::interface]
+trait IERC3525MetadataLegacy<TContractState> {
+    fn contractUri(self: @TContractState) -> felt252;
```

Comment:
in the cairo zero version I use `contractURI`. We might need to keep the same selector.